p2p/pre2p/telemetry/metrics.go

```diff
@@ -0,0 +1,20 @@
+package pre2p_telemetry
+
+const (
+	// How to define metrics in this file?
+	//
+	// Must define metrics using following template: MetricName_MetricType_Attribute
+	// e.g: type= timeseries, name=nodes_alive_counter, attribute=name
+	// will result in: NODES_COUNT_COUNTER_TIME_SERIES_NAME="nodes_alive_counter"
+	// you can also add: NODES_COUNT_COUNTER_TIME_SERIES_DESCRIPTION="for your description"
```

Comment:
Copy-pasta other comment: Only have this comment in one place (i.e. the telemetry readme)